MANIFEST.in

```diff
@@ -25,3 +25,4 @@ recursive-include openerp *.xml
 recursive-include openerp *.yml
 recursive-exclude * *.py[co]
 recursive-exclude * *.hg*
+recursive-include openerp/cli/templates *
```

Comment:
A [manifest template command](https://docs.python.org/2/distutils/sourcedist.html#commands) which simply includes all the files under the provided directory, without needing a filter.

So `graft $directory` should be more or less identical to `recursive-include $directory *`.